extensions/PhabBugz/lib/Revision.pm

```diff
@@ -0,0 +1,280 @@
+ # This Source Code Form is hasject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::Extension::PhabBugz::Revision;
+
+use 5.10.1;
+
+use Moo;
```

Comment:
since we're not using moo here this could go away -- but I could also Moo-sify it pretty easily.